pkg/providers/instancetype/suite_test.go

```diff
@@ -231,8 +228,6 @@ var _ = Describe("Instance Types", func() {
 			"topology.ebs.csi.aws.com/zone": "test-zone-1a",
 		}
 
-		// Ensure that we're exercising all well known labels
-		Expect(lo.Keys(nodeSelector)).To(ContainElements(append(v1alpha5.WellKnownLabels.UnsortedList(), lo.Keys(v1alpha5.NormalizedLabels)...)))
```

Comment:
Why are we removing this check in the suite_test? I think combined instance type labels and individual instance type labels should both make sure we're exercising all well known labels.

Reply:
Removing since there's no way to ensure that we are exercising all the labels through combination anymore since there's no instance type that will satisfy all the labels that we need to test.

I think ensuring we are testing all the labels on the individual label testing should be sufficient to ensure that we make sure that all the labels work

Reply:
Ah makes sense, I'd still like to make sure that this fails in the case of a new WellKnownLabel that's added, but not included in this test. Can we check that all the well known labels are used and either the set of GPU/Accelerator Labels are used? Something like
```
Expect(lo.Keys(nodeSelector)).To(ContainElements(append(v1alpha5.WellKnownLabels.UnsortedList().WithoutGPULabels(), lo.Keys(v1alpha5.NormalizedLabels)...)))
Expect(lo.Keys(nodeSelector)).To(ContainElements(append(v1alpha5.WellKnownLabels.UnsortedList().OnlyGPULabels(), lo.Keys(v1alpha5.NormalizedLabels)...)))

```